Resources/Prototypes/Entities/Objects/Devices/door_remote.yml

```diff
@@ -173,6 +175,8 @@
       color: "#2eba22"
     - state: door_remotescreencolour
       color: "#22871a"
+  - type: DoorRemote
+    ExtendedByID: true
```

Comment:
Lowercase e otherwise this will drop a silent error and not work (there's already a PR to fix this)